src/Swarm/Game/Value.hs

```diff
@@ -1,6 +1,7 @@
 -----------------------------------------------------------------------------
 -----------------------------------------------------------------------------
 {-# LANGUAGE GADTs #-}
+{-# LANGUAGE PatternSynonyms #-}
```

Comment:
Is this still necessary when you have `VInj s v`?

Reply:
Nope, good catch. Fixed now.